### PR TITLE
Enhancement/add ts

### PR DESCRIPTION
**Pull request by VictorBurgosK**

### CODE REVIEW

File: src/partner/modules/EventList/EventList.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+const EventList: React.SFC<IEventListProps> = props => {
+  if (props.events.length === 0) {
+    return <p>there are not events to show</p>;
+  }
+
+  return (
+    <EventsContainer>
+      {props.events.map(e => (
+        <EventListItem eventInfo={e} />
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventList.tsx]: `key` prop missing

File: src/partner/modules/EventList/EventListContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  isLoading: boolean;
+}
+
+class EventListContainer extends React.Component<{}, IEventListContainerState> {
+  state = {
+    events: [],
+    isLoading: false
+  };
+
+  public componentDidMount() {
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListContainer.tsx]: favor `async/await` over `promises`

File: src/partner/modules/EventList/EventListContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    this.setState({ isLoading: true });
+    eventService.getAll().then(events => {
+      events.sort(function(a, b) {
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListContainer.tsx]: extract into a separate function

File: src/partner/modules/EventList/EventListItem.tsx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+            }`}
+          </EventRowItem>
+        </RowData>
+      </RowDataContainer>
+      <RowDataContainer style={{ borderBottom: "0 none" }}>
+        <TableContainer>
+          <TableHead>
+            <tr>
+              <th>
+                <H2 style={{ textAlign: "left" }}>Tortas</H2>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+            }`}
+          </EventRowItem>
+          <EventRowItem>
+            {`${props.eventInfo.starTimeString} - ${
+              props.eventInfo.endTimeString
+            }`}
+          </EventRowItem>
+        </RowData>
+      </RowDataContainer>
+      <RowDataContainer style={{ borderBottom: "0 none" }}>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+        <TableContainer>
+          <TableHead>
+            <tr>
+              <th>
+                <H2 style={{ textAlign: "left" }}>Tortas</H2>
+              </th>
+              <th>
+                <H2 style={{ textAlign: "center" }}>Price</H2>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+              </th>
+              <th>
+                <H2 style={{ textAlign: "center" }}>Units</H2>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+              </th>
+              <th>
+                <H2 style={{ textAlign: "right" }}>Amount</H2>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+              </th>
+            </tr>
+          </TableHead>
+          <tbody>
+            <tr>
+              <td>
+                <EventRowItem style={{ textAlign: "left" }}>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+                  Tortas de Poc Chuc
+                </EventRowItem>
+              </td>
+              <td>
+                <EventRowItem style={{ textAlign: "center" }}>{`$${
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 11]
```diff
@@ -0,0 +1,174 @@
...
+                  props.eventInfo.pocChucTortaUnitPrice
+                }`}</EventRowItem>
+              </td>
+              <td>
+                <EventRowItem style={{ textAlign: "center" }}>{`${
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 12]
```diff
@@ -0,0 +1,174 @@
...
+                  props.eventInfo.pocChucTortaAmount
+                }`}</EventRowItem>
+              </td>
+              <td>
+                <EventRowItem style={{ textAlign: "right" }}>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 13]
```diff
@@ -0,0 +1,174 @@
...
+                  {`$${props.eventInfo.pocChucTotal}`}
+                </EventRowItem>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <EventRowItem style={{ textAlign: "left" }}>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 14]
```diff
@@ -0,0 +1,174 @@
...
+                  Tortas de Camarón
+                </EventRowItem>
+              </td>
+              <td>
+                <EventRowItem style={{ textAlign: "center" }}>{`$${
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 15]
```diff
@@ -0,0 +1,174 @@
...
+                  props.eventInfo.shrimpTortaUnitPrice
+                }`}</EventRowItem>
+              </td>
+              <td>
+                <EventRowItem style={{ textAlign: "center" }}>{`${
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 16]
```diff
@@ -0,0 +1,174 @@
...
+                  props.eventInfo.shrimpTortaAmount
+                }`}</EventRowItem>
+              </td>
+              <td>
+                <EventRowItem style={{ textAlign: "right" }}>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 17]
```diff
@@ -0,0 +1,174 @@
...
+                  {`$${props.eventInfo.shrimpTotal}`}
+                </EventRowItem>
+              </td>
+            </tr>
+          </tbody>
+          <TableFoot>
+            <tr>
+              <td>
+                <H2 style={{ textAlign: "left" }}>Total</H2>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/EventList/EventListItem.tsx [hunk 18]
```diff
@@ -0,0 +1,174 @@
...
+              </td>
+              <td />
+              <td />
+              <td>
+                <EventRowItem style={{ textAlign: "right" }}>
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid `inline-styles`

File: src/partner/modules/Header/Header.tsx [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+interface IHeaderProps {
+  title: string;
+}
+
+const Header: React.SFC<IHeaderProps> = props => {
+  return (
+    <header>
+      <div
+        className="header-container"
+        style={{
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `inline-styles`

File: src/partner/modules/Header/Header.tsx [hunk 20]
```diff
@@ -0,0 +1,97 @@
...
+          alignItems: "center",
+          padding: "5px 3px 25px 3px",
+          background: "linear-gradient(to right, #F8823D, #E83E5D)",
+          width: "100vw",
+          height: "6em"
+        }}
+      >
+        <div
+          className="header-logo-container"
+          style={{
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `inline-styles`

File: src/partner/modules/Header/Header.tsx [hunk 21]
```diff
@@ -0,0 +1,97 @@
...
+            width: "33.33%",
+            display: "flex",
+            flexDirection: "row",
+            alignItems: "center"
+          }}
+        >
+          <img
+            src={require("../../../images/poc-chef-logo.png")}
+            alt="Poc-Chef"
+            style={{
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `inline-styles`

File: src/partner/modules/Header/Header.tsx [hunk 22]
```diff
@@ -0,0 +1,97 @@
...
+              width: "100%",
+              height: "100%",
+              objectFit: "contain"
+            }}
+          />
+        </div>
+
+        <div
+          className="header-title-container"
+          style={{
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `inline-styles`

File: src/partner/modules/Header/Header.tsx [hunk 23]
```diff
@@ -0,0 +1,97 @@
...
+            style={{
+              width: "100%",
+              height: "100%",
+              objectFit: "contain"
+            }}
+          />
+        </div>
+
+        <div
+          className="header-title-container"
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `clasNames`

File: src/partner/modules/Header/Header.tsx [hunk 24]
```diff
@@ -0,0 +1,97 @@
...
+          justifyContent: "space-around",
+          alignItems: "center",
+          padding: "5px 3px 25px 3px",
+          background: "linear-gradient(to right, #F8823D, #E83E5D)",
+          width: "100vw",
+          height: "6em"
+        }}
+      >
+        <div
+          className="header-logo-container"
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `clasName`

File: src/partner/modules/Header/Header.tsx [hunk 25]
```diff
@@ -0,0 +1,97 @@
...
+            height: "100%",
+            width: "33.33%",
+            display: "flex",
+            flexDirection: "row",
+            alignItems: "center",
+            justifyContent: "center"
+          }}
+        >
+          <h1
+            style={{
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `inline-styles`

File: src/partner/modules/Header/Header.tsx [hunk 26]
```diff
@@ -0,0 +1,97 @@
...
+              color: "white",
+              fontSize: "16px"
+            }}
+          >
+            Current Events
+          </h1>
+        </div>
+        <div
+          className="header-user-icon-container"
+          style={{
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `inline-styles`

File: src/partner/modules/Header/Header.tsx [hunk 27]
```diff
@@ -0,0 +1,97 @@
...
+            flexDirection: "row",
+            alignItems: "center",
+            justifyContent: "flex-end"
+          }}
+        >
+          <img
+            src={require("../../../images/user-logo.png")}
+            alt="Poc-Chef"
+            style={{
+              width: "2.5em",
```
Comment (mishelashala) [inline src/partner/modules/Header/Header.tsx]: avoid `inline-styles`

**Review by mishelashala — CHANGES_REQUESTED** [review]

File: src/partner/modules/EventList/EventListContainer.tsx (hunk 3 above)
Comment (VictorBurgosK) [reply src/partner/modules/EventList/EventListContainer.tsx]: Do you want the "sort function" as a service or some like that?

**Review by mishelashala — APPROVED** [review]

**Pull request merged**